tests/t5/test_modeling_tf_t5.py

```diff
@@ -227,23 +227,6 @@ def create_and_check_t5_decoder_model_past_large_inputs(
         # test that outputs are equal for slice
         tf.debugging.assert_near(output_from_past_slice, output_from_no_past_slice, rtol=1e-3)
 
-    def create_and_check_t5_xla_generate(self, config, input_ids, *args):
```

Comment:
same -> think XLA should be run at least once on CPU for every PR. Let's add a skipif with a TODO statement?